docs/sources/mobile-app/_index.md

```diff
@@ -93,6 +93,11 @@ Intended for critical alerts. Device-specific settings may impact the functional
 >**Note**: Android users may need to review their device settings to ensure the Grafana OnCall mobile app is authorized to bypass Do Not Disturb.
 > In your device settings, allow the Android Critical Message Channel notification channel to Override Do Not Disturb.
 
+>**Note**: Unfortunately, Android does not support critical alerts. As a result, we need to use a workaround in order to override the notification volume. 
+> This means that the app will change your notification volume, and we are unable to revert it automatically. 
+> If you prefer to avoid this behavior, go to the app Settings -> Configure Sounds, and disable the `Override System Volume` switch.
```

Comment:
```suggestion
> If you prefer to avoid this behavior, go to the app `Settings` -> `Configure Sounds`, and disable the `Override System Volume` switch under `Important notifications`.
```